### DISCUSSION TITLE
超級好用的ChatGPT

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS3073-C/Python-Community-GS3073-C#526] [General]
ChatGPT是由[OpenAI]開發的一個[人工智慧][聊天機器人]程式，於2022年11月推出。該程式使用基於[GPT-3.5]架構的[大型語言模型]並通過[強化學習]進行訓練。它以對話方式進行互動。在推廣的期間中，所有人可以免費註冊，並在登入後後免費使用 ChatGPT 實現與 AI 機器人對話。

**Comment by gnsJhenJie** [comment]
網址在這: https://chat.openai.com/chat
ChatGPT 目前還沒有發paper或open source，所以大家想自己建一個的話可能還要等等

**Comment by todoroki0910** [comment]
這東西真的能用來寫小說

**Comment by awkward-willy** [comment]
近期它真的非常紅，也有許多人測試出奇奇怪怪的各種應用方式，甚至有人成功繞過它的安全機制騙得不符合規範的答覆，其實是相當有趣且新穎，但也是值得擔心的一項科技。

**Comment by williamyen0623** [comment]
好期待chatGPT的 open source，好想知道到底是怎麼做出這個神奇的酷東東

**Comment by limitx0** [comment]
和ChatGPT對話真的滿好玩的
雖然有時候會出錯

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Raywei0214** [comment]
我們都知道，只要有意義，那麼就必須慎重考慮。chatGPT必定會成為未來世界的新標準。但丁說過一句著名的話，生活於願望之中而沒有希望，是人生最大的悲哀。

**Comment by alvin1226** [comment]
上次做心得懶得寫，叫他幫我寫出來，自己在修飾一下，就可以交差了，這ai真棒
